modules/working-file-repository-service-impl/src/main/java/org/opencastproject/workingfilerepository/impl/WorkingFileRepositoryImpl.java

```diff
@@ -180,7 +180,7 @@ public void activate(ComponentContext cc) throws IOException {
     // Determine which collections should be garbage collected
     List<String> collectionsToCleanUp = null;
     String[] cleanupCollections = StringUtils.split(StringUtils.trimToNull(
-            cc.getBundleContext().getProperty(WORKING_FILE_REPOSITORY_CLEANUP_COLLECTIONS_KEY)));
+            cc.getBundleContext().getProperty(WORKING_FILE_REPOSITORY_CLEANUP_COLLECTIONS_KEY)),',');
```

Comment:
Could you add a space:

```suggestion
            cc.getBundleContext().getProperty(WORKING_FILE_REPOSITORY_CLEANUP_COLLECTIONS_KEY)), ',');
```